tests/backends/pulser_basic/test_configuration.py

```diff
@@ -70,5 +71,5 @@ def test_configuration_as_dict() -> None:
     assert not np.isclose(div, 0.0, rtol=1e-2, atol=1e-2)
 
     wrong_conf = {"wrong": "value"}
-    with pytest.raises(ValueError):
+    with pytest.raises(TypeError):
```

Comment:
different type here

Reply:
hmm, I had to change this back to `ValueError` to make the test pass, or did you change this because we should convert that thrown `ValueError` to a `TypeError`?

Reply:
interesting, on my end suddenly it started raising `TypeError` rather than ``ValueError`